README.md

```diff
@@ -39,6 +39,15 @@ We recommend installing NeMo in a fresh Conda environment.
     conda create --name nemo_tn python==3.8
     conda activate nemo_tn
 
+Install PyTorch using their `configurator <https://pytorch.org/get-started/locally/>`_. 
```

Comment:
could you add that this is optional - for the hybrid system only

Reply:
done